src/content/configuration/resolve.md

```diff
@@ -414,6 +414,35 @@ module.exports = {
 ```
 
 
+### `resolve.preferRelative`
+
+`boolean`
+
+When enabled, webpack would prefer to resolve module requests as relative requests than modules from `node_modules` directories.
```

Comment:
```suggestion
When enabled, webpack would prefer to resolve module requests as relative requests instead of using modules from `node_modules` directories.
```